src/devices/Si7021/samples/README.md

```diff
@@ -0,0 +1,36 @@
+# Example of Si7021
```

Comment:
Example for

Reply:
I'd change to `SI7021 - samples` though

Reply:
Aha, I found out that you didn't take a close look at README in my previous PRs. 😄

Reply:
I usually just skim through READMEs if they have the right structure - they are easier to change after they are added vs the code which someone can start to depend on 😄 I'll try to look more but I'm currently the only one checking PRs and it takes quite a bit of my time